graylog2-web-interface/src/integrations/aws/cloudwatch/SidebarPermissions.tsx

```diff
@@ -14,112 +14,21 @@
  * along with this program. If not, see
  * <http://www.mongodb.com/licensing/server-side-public-license>.
  */
-import React, { useState } from 'react';
-import styled, { css } from 'styled-components';
+import React from 'react';
 
-import { Panel } from 'components/bootstrap';
-import useFetch from 'integrations/aws/common/hooks/useFetch';
-import { ApiRoutes } from 'integrations/aws/common/Routes';
-import Icon from 'components/common/Icon';
-
-type PoliciesProps = {
-  title: string;
-  note: string;
-  policy: any;
-};
-
-function Policies({ title, note, policy }: PoliciesProps) {
-  const [opened, setOpened] = useState(false);
-
-  const toggleOpen = () => {
-    setOpened(!opened);
-  };
-
-  return (
-    <div>
-      <Header onClick={toggleOpen}>
-        <HeaderContent>
-          <Title>
-            {opened ? 'Hide' : 'Show'} {title}
-          </Title>
-          <Note>{note}</Note>
-        </HeaderContent>
-
-        <IconContainer $opened={opened}>
-          <Icon name="arrow_right_alt" size="2x" />
-        </IconContainer>
-      </Header>
-
-      <Policy opened={opened}>{JSON.stringify(policy, null, 2)}</Policy>
-    </div>
-  );
-}
+import {Panel} from 'components/bootstrap';
+import {ExternalLink} from 'components/common';
 
 export default function SidebarPermissions() {
-  const [permissionsStatus] = useFetch(ApiRoutes.INTEGRATIONS.AWS.PERMISSIONS);
 
   return (
     <Panel bsStyle="info" header={<span>AWS Policy Permissions</span>}>
       <p>
-        Please verify that you have granted your AWS IAM user sufficient permissions. You can use the following policies
-        for reference.
+        Please refer to the{' '}
+        <ExternalLink href="https://go2docs.graylog.org/current/getting_in_log_data/aws_kinesis_cloudwatch_input.html">official documentation</ExternalLink>
```

Comment:
One small change I am still working on: Instead of referring to the docs site by a full URL, I am checking if we can use an alias to avoid hard-coding the full URL here using one of these aliases:
https://github.com/Graylog2/graylog2-server/blob/206c6b34d67a28d91fa77f68d0df56b5d6140046/graylog2-web-interface/src/util/DocsHelper.ts#L20-L60